docs/source/Account/Run/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _Run:
```

Comment:
in index.rst it is not necessary to create links, remove all .._Text:

Reply:
I removed the .. _text: from the index.rst file.